Add and update configs for Novo Nordisk HPCs

VincentAaskov opened the pull request:
---
name: Add and update configs for Novo Nordisk HPCs 
about: Updating the existing config for Marjorie as well as adding missing configs for Dirac and Lovelace, all tested in August 2025 after new in-house hosting of docker images and updated modules.
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any -> None

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
- [ ] OPTIONAL: Add your custom profile path and GitHub user name to `.github/CODEOWNERS` (`**/<custom-profile>** @<github-username>`)

<!--
If you require/still waiting for a review, please feel free to request a review from @nf-core/maintainers

Please see uploading to`nf-core/configs` for more details:
https://github.com/nf-core/configs#uploading-to-nf-coreconfigs

Thank you for contributing to nf-core!
-->

jfy133 commented on docs/dirac.md:
```diff
@@ -0,0 +1,35 @@
+Using this profile, Docker image(s) containing required software(s) will be downloaded
+from the in-house Docker repository, and converted to `Apptainer` image(s) if needed before execution of the pipeline.
+
+## Below are non-mandatory information on e.g., required modules
```
Minor thing: `non-mandatory` and `required` are contradictory, maybe clariufy

VincentAaskov replied:
@jfy133 : Thank you - addressed with more clarity

jfy133 reviewed (approved):
nice and clean, LGTM @VincentAaskov !

FriederikeHanssen commented on conf/lovelace.config:
```diff
@@ -0,0 +1,36 @@
+    ]
+    executor     = 'slurm'
+    queue        = 'compute'
+    scratch      = true
+    maxRetries   = 2
+    scratch      = "/scratch/${USER}"
```
same here

FriederikeHanssen commented on conf/marjorie.config:
```diff
@@ -21,7 +18,7 @@ process {
     executor     = 'slurm'
     queue        = 'compute'
     scratch      = true
-    maxRetries   = 3
+    maxRetries   = 2
     scratch      = "/scratch/${USER}"
```
Noticed that the scratch value is also set twice here. 

`true` should tell Nextflow to use `/tmp` or whatever is set in `$TMPDIR` https://nextflow.io/docs/latest/reference/process.html#scratch, while setting an actual directory will use that. I assume you want the second one?

VincentAaskov replied:
Addressed by only keeping `scratch = "/scratch/${USER}"`

VincentAaskov commented:
@jfy133 and @FriederikeHanssen : Thank you for your review and comments. I've addressed your comments, but hasn't resolved them as I expect you would like to give your explicit approval of my fixes. However, even if resolved I'm not able to merge the branch as the `Lint tools code formatting` check fails. I honestly can't see what it fails on. I've tried running `prettier` on the offending files (`docs/dirac.md` and `docs/lovelace.md`) but this doesn't change anything. From the log I can't see what is actual changed by `prettier`. Please advice on how to proceed.

jfy133 commented:
We have a trick for that (that normally works):

@nf-core-bot fix linting

jfy133 reviewed (approved):
LGTM to me!
